packages/deno/src/integrations/deno-cron.ts

```diff
@@ -1,61 +1,65 @@
-import { withMonitor } from '@sentry/core';
-import type { Integration } from '@sentry/types';
-import type { DenoClient } from '../client';
+import { convertIntegrationFnToClass, getClient, withMonitor } from '@sentry/core';
+import type { Client, IntegrationFn } from '@sentry/types';
 import { parseScheduleToString } from './deno-cron-format';
 
 type CronOptions = { backoffSchedule?: number[]; signal?: AbortSignal };
 type CronFn = () => void | Promise<void>;
 // Parameters<typeof Deno.cron> doesn't work well with the overloads 🤔
 type CronParams = [string, string | Deno.CronSchedule, CronFn | CronOptions, CronFn | CronOptions | undefined];
 
+const INTEGRATION_NAME = 'DenoCron';
+
+const SETUP_CLIENTS: Client[] = [];
```

Comment:
This means that clients dont get GC'd - I wonder if we should track with a WeakMap instead?